components/expenses/graphql/fragments.ts

```diff
@@ -168,6 +168,28 @@ export const expenseHostFields = gql`
   ${accountingCategoryFields}
 `;
 
+export const expenseValuesByRoleFragment = gql`
+  fragment ExpenseValuesByRoleFragment on ExpenseValuesByRole {
+    id
+    submitter {
+      accountingCategory {
+        ...AccountingCategoryFields
+      }
+    }
+    accountAdmin {
+      accountingCategory {
+        ...AccountingCategoryFields
+      }
+    }
+    hostAdmin {
+      accountingCategory {
+        ...AccountingCategoryFields
+      }
+    }
+  }
+  ${accountingCategoryFields}
+`;
```

Comment:
Added `valuesByRole` here to consolidate the experience between the expenses list and expense page. Also simplified props by passing this value directly to `ExpenseCategorySelect`